src/freezing/freezing_main.py

```diff
@@ -124,3 +118,31 @@ def _add_last_frozen_column(
     last_frozen = f"{todays_date}_v{str(run_id)}"
     frozen_df["last_frozen"] = last_frozen
     return frozen_df
+
+
+def read_frozen_csv(config: dict,
+                    read_csv: Callable) -> pd.DataFrame:
```

Comment:
small styling?
I think the pre-commit would force this to be something like as follow

```python
def read_frozen_csv(
    config: dict,
    read_csv: Callable,
) -> pd.DataFrame:
```